File: code-java/aula1-everis/src/main/java/br/net/uno/HelloEndPoint.java
package br.net.uno;

import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;

@Controller("/hello")
public class HelloEndPoint {

    @Get("/{name}")
    public String hello(String name) {
        return "Ola, "+ name + " !";
    }
}

File: code-kotlin/aula1-everis/src/main/kotlin/br/net/uno/HelloEndPoint.kt
package br.net.uno

import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get

@Controller("/hello")
class HelloEndPoint {

    @Get("/{name}")
    fun hello(name: String): String {
        return "Ola, $name voce esta agora em kotlin !"
    }

    @Get("/teste")
    fun getTeste() = "funcao de teste, micronaut continua rodando"

}